extensions/extension.xsl

```diff
@@ -76,6 +76,9 @@
       <p><strong>DO NOT IMPLEMENT!!!</strong></p>
       </xsl:if>
 
+      <xsl:comment>Help the community by recording statistics on availability of 3D APIs. Trick to prevent XSL processor from creating an illegal self-closing iframe tag.</xsl:comment>
+      <iframe src="https://web3dsurvey.com/collector-iframe.html" style="width: 1px; height: 1px;" frameborder="0"><xsl:text>&#160;</xsl:text></iframe>
```

Comment:
how about

```html
<iframe src="https://web3dsurvey.com/collector-iframe.html" width="1" height="1" style="border: none;">
```

IIUC, the code as is is asking for a 300x150 iframe to be scaled to 1x1 where as the suggested code is asking for a 1x1 iframe. Also [`frameborder="0"` is deprecated](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#frameborder).

Reply:
AFAICT the width/height attributes do the same thing as the CSS, on iframes. So could just as well be all CSS.

Reply:
You're right! Thanks!